### PR TITLE
Psse Support Version 32

### PR DESCRIPTION
**Pull request by marqueslanauja**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the current behavior?** *(You can also link to an open issue here)*
Only versions 33 and 35 were supported.

**What is the new behavior (if this is a feature change)?**
Versions 32, 33 and 35 are supported.

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

**Comment by marqueslanauja** [comment]
We prefer to explicitly see the blocks of each version even if code is duplicated.

**Review by zamarrenolm — APPROVED** [review]
Sonar warnings about duplicated code refer to the data blocks defined for every specific PSSE version supported. I think it is preferable to keep the current code, even it is marked as duplicated, because it reflects the structure of the file in each PSSE revision and is easy to check against published documentation.

File: psse/psse-model/src/main/java/com/powsybl/psse/model/pf/io/PowerFlowRawData32.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
```
Comment (MioRtia) [inline psse/psse-model/src/main/java/com/powsybl/psse/model/pf/io/PowerFlowRawData32.java:2]: *Copyright 2021

**Review by MioRtia — APPROVED** [review]

File: psse/psse-model/src/main/java/com/powsybl/psse/model/pf/io/PowerFlowRawData32.java (hunk 1 above)
Comment (marqueslanauja) [reply psse/psse-model/src/main/java/com/powsybl/psse/model/pf/io/PowerFlowRawData32.java]: Done

**Review by geofjamg — APPROVED** [review]

**Pull request merged**